.ci/aws/Jenkinsfile

```diff
@@ -199,26 +188,33 @@ pipeline {
                     def test_type = "--test-type pr"
                     def build_type = "--aws-ofi-nccl-build-type debug"
                     def pr_num = "--test-aws-ofi-nccl-pr $env.CHANGE_ID"
-                    def test_list = "--test-list test_nccl_test test_ofi_nccl_functional"
                     def nccl_test_iter = "--test-aws-ofi-nccl-nccltest-iterations 5"
                     def efa_installer = "--use-prebuilt-ami-with-efa-installer true"
-                    def base_args = "${efa_installer} ${nccl_version} ${timeout} ${cluster_type} ${test_target} ${test_type} ${build_type} ${pr_num} ${test_list} ${nccl_test_iter}"
+
+                    def persistent_manual_cluster_addl_args = " --keep-cluster --skip-fixture-setup --skip-health-checks --use-existing-installer --cleanup-pf-directory"
+                    def container_addl_args = " --test-in-containers-on-ec2"
+
+                    def base_args = "${efa_installer} ${nccl_version} ${timeout} ${cluster_type} ${test_target} ${test_type} ${build_type} ${pr_num} ${nccl_test_iter} ${persistent_manual_cluster_addl_args}"
 
                     def num_instances = 4
                     def p3dn_lock_label = "p3dn-1-4node"
                     def p3dn_region = "ap-northeast-1"
```

Comment:
I think you can reduce the number of arguments here drastically... getting rid of ODCR, and region ;)